src/components/MemberArea/MemberArea.js

```diff
@@ -124,6 +125,7 @@ function MemberArea({ onOpenModal }) {
       ) : (
         isDesktop && <LoginNavigation />
       )}
+      <GlobalStyle />
```

Comment:
This is a bit confusing.  Isn't `<GlobalStyle>` already available to this component since it's part of the render process of `Me.js`?

Reply:
Yes, it's confusing. No, it's not in the render process of me because this component is in the front side, not the backoffice.
Maybe it will be a better idea to put it in [`<App />`](https://github.com/Coding-Coach/find-a-mentor/blob/4349034bc427f02871b1f5b87eb62c5da44592c2/src%2Fcomponents%2FApp%2FApp.js)

Reply:
Ahh, I see, ok.  If it's placed in `<App />` should it be _removed_ from `Me.js`?

Reply:
I will leave this for another ticket to pull this out of both places.